wri/gfw
# [FLAG-693] widget tree cover density

## Overview


### Story | Acceptance criteria

As a user, I want to view average tree cover of area vs. percent of area with tree cover so that I can understand diversity of tree densities within my area of interest

Given that:
- The user is on the dashboard page
- The “Summary”, or “Land Cover” tab is active
- The user has selected: 
  - an admin boundary (adm 0-2) or country, region, and subregion
  - selected a custom area (with a “saved” status)

#### Widget title

- There is a title that reads: 
  - Admin areas: “Tree cover density in [area]”

#### Widget sentence 

- There is a sentence that reads 
  - “[Area] had an average tree cover of [percent canopy cover] in 2020.”

#### Graph

- Below the widget sentence, there is a bar graph wherein: 
  - The x axis is labeled “Tree cover (%)” and bars represent different tree cover percentages - 10, 20, 30, 40, 50, 60, 70, 80, 90, and 100%. 

  - The y axis is labeled “Land (in hectares)” and rows are labeled 0, 20,000, 40,000, 60,000, 80,000, 100,000, 120,000, and 140,000 (amounts will be in hectares, longform).  

### Widget settings 

- There is a button with a map icon:
  - When hovered on it reads “Show on map“
  - When clicked, it shows the tree cover gain layer on the dashboard map
  
- There is a button with an “i” icon:
  - When hovered on it reads “Learn more about the data“
  - When clicked, it opens the metadata modal (metadata url linked below)

- There is a button with a share icon:
  - When hovered on it reads “Share or embed this widget“
  - When clicked, it shows the sharing modal

- There is a button in the widget with a gear icon:
  - When hovered on it reads “Filter and customize the data”
  - When clicked, it shows the settings panel for this widget

- There is a selector for land category, for which the options are:  
  - Alliance for zero extinctions
  - Indigenous and Community Lands
  - Key Biodiversity 
  - Areas logging concessions  
  - Mining concessions  
  - Oil palm concessions  
  - Wood fiber concessions
  - Protected areas 


## Demo

![Screenshot 2023-04-19 at 12 38 17](https://user-images.githubusercontent.com/23243868/233127792-46ead3ac-a19a-4da8-b359-1113e600ee0c.png)

## Notes

If applicable: ancilary topics, caveats, alternative strategies that didn't work out, etc.

## Testing

- Open the dashboard and select any country and/or subregions.
- Alternatively you can open directly using [this link](https://gfw-staging-pr-4518.herokuapp.com/dashboards/country/BRA/1/?category=land-cover)
- Check the widget layout, position and others acceptance criteria.
